### ISSUE TITLE
Full finetuning or LoRA for ISE

### ISSUE THREAD

**Issue by lindsey98** [tongwu2020/ISE#3]
Hi Tong,
I noticed that in the paper, ISE is using full fine-tuning. While you also provided a LoRA (Low-Rank Adaptation) tuning configuration, it seems to be for scenarios without ISE (please correct me if my understanding is wrong).

Could you clarify whether full fine-tuning is specifically recommended for ISE? 

I ask this question because I have tried LoRA with target modules as "q_proj", "v_proj", "o_proj", as well as the ISE embedding layer. However, I did not get satisfactory results in terms of utility and robustness. So, I suspect that the addition of the ISE embedding layer might significantly alter the input distribution, potentially making full fine-tuning necessary. If you have experience with combining ISE and LoRA, I would greatly appreciate your suggestions and insights.

**Comment by tongwu2020** [comment]
Hi Lindsey, 

I would recommend using full fine-tuning and starting from the base model. If you want to use ISE with LORA, I think you should initialize the ISE with a very small value and use a different learning rate for the ISE layer. I learned from another paper (not public yet) that ISE can work with LORA.

**Comment by lindsey98** [comment]
Thank you for your suggestions!
